airbyte-integrations/connectors/source-bing-ads/source_bing_ads/bulk_streams.py

```diff
@@ -11,17 +11,20 @@
 from numpy import nan
 
 from airbyte_cdk.models import SyncMode
-from airbyte_cdk.sources.streams import IncrementalMixin
+from airbyte_cdk.sources.streams import CheckpointMixin
 from airbyte_cdk.sources.utils.transform import TransformConfig, TypeTransformer
 from source_bing_ads.base_streams import Accounts, BingAdsBaseStream
 from source_bing_ads.utils import transform_bulk_datetime_format_to_rfc_3339
 
 
-class BingAdsBulkStream(BingAdsBaseStream, IncrementalMixin, ABC):
+class BingAdsBulkStream(BingAdsBaseStream, CheckpointMixin, ABC):
     transformer: TypeTransformer = TypeTransformer(TransformConfig.DefaultSchemaNormalization | TransformConfig.CustomSchemaNormalization)
     cursor_field = "Modified Time"
     primary_key = "Id"
-    _state = {}
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._state = {}
```

Comment:
Tests like `test_bulk_stream_stream_state` failed locally for me even before updating the CDK version. I moved the state to an instance variable before updating the CDK version to have this test green and updated the CDK version afterward